diffdb/db_test.go

```diff
@@ -2,13 +2,15 @@ package diffdb
 
 import (
 	"math/big"
+	"os"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
-func TestInMemoryDb(t *testing.T) {
-	db, err := NewDiffDb("whatever")
+func TestDiffDb(t *testing.T) {
+	os.Remove("./test_diff.db")
```

Comment:
Other tests use `ioutil.Tempdir`, but also not that big of a deal
An example: https://github.com/ethereum-optimism/go-ethereum/blob/930c9f1381bb304496b036bab8f51899d1e63c71/core/bench_test.go#L148